src/index.js

```diff
@@ -4,10 +4,7 @@ function knit(arg, item) {
   if (!Array.isArray(arg)) {
     throw TypeError(`I can't knit with ${type(arg)}`)
   }
-  if (!arg.length) {
-    throw TypeError(`I can't knit with an empty array!`)
-  }
-  if (arg.length === 1) return arg
+  if (arg.length <= 1) return arg
```

Comment:
I know it’s not enforced rn, but please don’t use block statement here for readability. thx

Reply:
I changed it. Btw originally the return was on the same line.